HorizonEngine/Platform/OpenGL/GLFrameBuffer.cpp

```diff
@@ -27,8 +28,8 @@ namespace Hzn
 			uint32_t samples,
 			uint32_t attachmentNumber,
 			uint32_t id,
-			uint32_t width,
-			uint32_t height,
+			int32_t width,
```

Comment:
Why did you change from uint32_t to int32_t?

Reply:
Hi,
This was because when the values that width and height received were negative, Then when the bits are interpreted as unsigned integers, -1 is intepreted as UINT_MAX (because last bit is for sign, in signed integers).